### ISSUE TITLE
It started listing things for sale automatically and I lost at least one skin!

### ISSUE THREAD

**Issue by ShiftlessDrifter** [Nuklon/Steam-Economy-Enhancer#190]
I installed this script for Tampermonkey yesterday, January 1. Today, I was looking at the Steam Market and noticed I had a lot of sell orders for some reason. See screenshot.

![image](https://github.com/Nuklon/Steam-Economy-Enhancer/assets/133805836/34117a12-9b07-4d9b-8259-2ed21b88b6d7)

I did not create these listings. I am not trying to sell any skins. I immediately cancelled all of the listings, but when I refreshed my history I see that one item sold for $0.99. I was able to buy it back for $1.05 -- so a six cent loss and not a big deal.

But now I'm worried about my Steam account. How was this script/extension able to market and sell _my own skins_ and **without my permission?** Do I need to change the password on my Steam account? (I have 2FA and haven't been alerted by attempted break-ins). I'm so confused. I have since disabled Tampermonkey and uninstalled this script.

**Comment by woctezuma** [comment]
Maybe:

![Picture](https://i.imgur.com/Lb1ECeM.png)

**Comment by ShiftlessDrifter** [comment]
No, only 1 was a duplicate. All the rest, including the one that sold, was my only one.
